stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/TextFieldUI.kt

```diff
@@ -41,11 +50,21 @@ import androidx.compose.ui.semantics.semantics
 import androidx.compose.ui.text.AnnotatedString
 import androidx.compose.ui.text.input.ImeAction
 import androidx.compose.ui.unit.dp
+import com.stripe.android.core.Logger
+import com.stripe.android.uicore.BuildConfig
 import com.stripe.android.uicore.R
 import com.stripe.android.uicore.stripeColors
 import kotlinx.coroutines.delay
 import kotlinx.coroutines.launch
 
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+val LocalAutofillEventReporter = staticCompositionLocalOf<(String) -> Unit> {
+    {
```

Comment:
Is this an extra set of parenthesis? I'm assuming it could also be part of the method. Maybe it would make it more clear if we added a named parameter here, rather than using `it` below.